tests/sentry/utils/performance_issues/test_performance_detection.py

```diff
@@ -434,6 +434,7 @@ def test_reports_metrics_on_uncompressed_assets(self, incr_mock):
                     "consecutive_db": False,
                     "large_http_payload": False,
                     "consecutive_http": False,
+                    "consecutive_http_ext": False,
                     "slow_db_query": False,
                     "render_blocking_assets": False,
                     "n_plus_one_db": False,
```

Comment:
Aside / Cleanup note (doesn't have to be this PR). We could prob re-arrange this asset to just check for the `uncompresset_asset` tag in the call instead of this growing tags value (that needs to be updated every time we update this).